spec/student_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rspec'
+require_relative '../student'
+
+describe Student do
+  let(:student) { Student.new(16, 'Alice', parent_permission: true, classroom: 'Math') }
+
+  context 'attributes' do
+    it 'has an age' do
+      expect(student.age).to eq(16)
+    end
+
+    it 'has a name' do
+      expect(student.name).to eq('Alice')
+    end
+
+    it 'has parent permission' do
+      expect(student.can_use_services?).to be true
+    end
+
+    it 'has a classroom' do
+      expect(student.classroom).to eq('Math')
+    end
+  end
+
+  context 'play_hooky' do
+    it 'returns a shrug emoticon' do
+      expect(student.play_hooky).to eq('¯\(ツ)/¯')
+    end
+  end
+end
```

Comment:
- [ ] [REQUIRED] Kindly add a unit test for the `classroom` method in the student class as well. For example:

``` ruby
describe Student do
  let(:student) { Student.new(16, 'Alice', parent_permission: true, classroom: 'Math') }
  let(:classroom) { Classroom.new('Math') }

  context 'classrooms=' do
    it 'assigns the classroom and adds the student to the classroom' do
      student.classrooms = classroom
      expect(student.classroom).to eq(classroom)
      expect(classroom.students).to include(student)
    end
  end
end
```